### DISCUSSION TITLE
Running 'Python368.efi chipsec_main.py -m tools.smm.smm_ptr -a fuzzmore 0x00:0x01' with Chipsec v1.9.1 failed with an exception.

### DISCUSSION THREAD

**Discussion by girishd22** [chipsec/chipsec#1603] [Q&A]
Running 'Python368.efi chipsec_main.py -m tools.smm.smm_ptr -a fuzzmore smic_start:smic_end' command in UEFI with Chipsec v1.9.1 failed with an exception "[-] ERROR: Exception occurred during chipsec.modules.tools.smm.smm_ptr.run(): 'function takes exactly 4 arguments (3 given)'"

The same command worked with Chipsec v1.8.7. Can anyone let me know what has changed?

Thanks,
Girish

**Comment by npmitche** [comment]
Can you try again with the changes from this PR: https://github.com/chipsec/chipsec/pull/1605

**Reply by girishd22** [reply]
Hi Nathaniel Mitchell,
I am executing this command from UEFI shell. Made the changes as in the PR: https://github.com/chipsec/chipsec/pull/1605 in chipsec/helper/efi/efihelper.py. Still getting the same error message. Is there any other file changes that needs to be done?

Thanks,
Girish

**Reply by npmitche** [reply]
Looks like I forgot to update an instance of writemem. 😨 I updated the PR, could you try again?

**Reply by girishd22** [reply]
Hi Nathaniel Mitchell,
Great!
 It works now. Thank you so much for your quick response. Appreciate it!
Btw, when will this change go into the main branch? 

Thanks,
Girish

**Reply by npmitche** [reply]
Hopefully in the next week or two.
